README.md

```diff
@@ -1,6 +1,202 @@
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 CRT Mobile Menu <a name="about-project"></a>
+
+**CRT Mobile Menu** is a portfolio website that allows users to view my projects.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+  </ul>
+</details>
+
+
+<!-- Future -->
+
+### Key Features <a name="key-features"></a>
+
+- Dynamic mobile menu
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+  node.js
+  git
+  JavaScript
+  Webpack
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:KanzaTahreem/CRT_Mobile_Menu.git
+```
+### Install
+
+Install this project with:
+
+```sh
+  cd CRT_Mobile_Menu
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  Open live server
+```
+### Run tests
+
+To run tests, run the following command:
+
+```
+  npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@KanzaTahreem](https://github.com/KanzaTahreem)
+- Twitter: [@kanza_tahreem](https://twitter.com/kanza_tahreem)
+- LinkedIn: [Kanza Tahreem](https://www.linkedin.com/in/kanzatahreem/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- Add dynamic project section
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]([../../issues/](https://github.com/KanzaTahreem/CRT-Mobile-Menu/issues)).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
```

Comment:
## [OPTIONAL] :grey_exclamation: :grey_exclamation: :grey_exclamation:

**Please note that this optional suggestion will not stop your work from being approved.**

- [ ] As a professional developer, having good communication skills is part of what recruiters watch out for, so they can be sure of maintaining their company's culture where communication of ideas, intent or actions are without or with distortions, ambiguity or incoherence to achieve their goals, and avoid some costs. I think writing complete sentences where necessary  all the time would be an added advantage to your brand.

- Here for example, one can complete this as:
`````sh
 If you like this project, please consider giving it a star. Thank you for the support 🙏